src/main/resources/db/migration/common/V2025.01.07.0002__add_pre_release_field.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE review
```

Comment:
Accidentally added 0001 of this file in a previous commit, which was merged. Thought about fixing the database etc but thought it best to just create an 0002 with the change I wanted in it.